src/harness/values.yaml

```diff
@@ -107,11 +107,19 @@ global:
 
   # -- Istio Ingress Settings
   istio:
-    enabled: false
```

Comment:
why are we making the default value `true`

Reply:
updated.